# OrientationService.srv
std_msgs/Float32 goal
---
bool finish